pom.xml

```diff
@@ -227,15 +228,13 @@
         <apache.shiro.version>1.3.2</apache.shiro.version>
         <spark.version>2.5.5</spark.version>
         <sardine.version>5.7</sardine.version>
+        <xerces.version>2.11.0</xerces.version>
 
         <!-- gitflow-incremental-builder -->
         <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
         <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
         <gib.baseBranch>HEAD</gib.baseBranch>
         <gib.enabled>false</gib.enabled>
-        <commons-lang3.version>3.4</commons-lang3.version>
```

Comment:
Don't need commons-lang3 property anymore?

Reply:
It was in there twice :-)